mido/midifiles/meta.py

```diff
@@ -11,6 +11,7 @@
      - expose _key_signature_encode/decode?
 """
 from __future__ import print_function, division
+from bidict import bidict
```

Comment:
I can definitely remove this if need be. I felt at least a little weird adding a dependancy for it.

This is a great package, especially for this "encode/decode" thing you have going on here with the dictionaries.

The bidict basically can have keys as values, and values as keys, so you can access the "inverse" of the dictionary with "my_bidict.inv", similar to what was being done in "reverse_table".

Again, happy to revert out this bidict change, just wanted to show what it was capable of!